dotnet/samples/Demos/ModelContextProtocolPlugin/Program.cs

```diff
@@ -70,3 +71,23 @@
 var prompt = "Summarize the last four commits to the microsoft/semantic-kernel repository?";
 var result = await kernel.InvokePromptAsync(prompt, new(executionSettings)).ConfigureAwait(false);
 Console.WriteLine($"\n\n{prompt}\n{result}");
+
+// Define the agent
+ChatCompletionAgent agent =
+new()
+{
+    Instructions = "Answer questions about GitGub repositories.",
+    Name = "GitHubAgent",
+    Kernel = kernel,
+    Arguments = new KernelArguments(new PromptExecutionSettings() { FunctionChoiceBehavior = FunctionChoiceBehavior.Auto() }),
+};
+
+/// Create the chat history thread to capture the agent interaction.
+AgentThread thread = new ChatHistoryAgentThread();
+
+// Respond to user input, invoking functions where appropriate.
+Console.WriteLine("\n\nResponse from GitHubAgent:");
+await foreach (ChatMessageContent response in agent.InvokeAsync("Summarize the last four commits to the microsoft/semantic-kernel repository?", thread))
+{
+    Console.WriteLine(response.Content);
+}
```

Comment:
Also consider the following.  I'm thinking of updating a lot of our current samples to use this pattern instead, since we don't expect more than one response.
```suggestion
ChatMessageContent response = await agent.InvokeAsync("Summarize the last four commits to the microsoft/semantic-kernel repository?", thread).FirstAsync();
Console.WriteLine(response.Content);
```